nautobot/docs/configuration/optional-settings.md

```diff
@@ -171,6 +171,30 @@ Celery broker URL used to tell workers where queues are located.
 
 ---
 
+## CELERY_WORKER_REDIRECT_STDOUTS
```

Comment:
nit here. Should we move these after `## CELERY_WORKER_PROMETHEUS_PORTS`? So it is somewhat alpha sorted and `2.0.0` changes are behind `1.5.10`?